CMakeLists.txt

```diff
@@ -37,11 +38,30 @@ if ($ENV{CLION_IDE})
 endif ()
 target_compile_definitions(llvm_interface INTERFACE ${LLVM_DEFINITIONS})
 
-if ($ENV{CLION_IDE})
-    message(STATUS "Using extra llvm include files as CLion can't follow WSL symlinks")
-    target_include_directories(llvm_interface INTERFACE "/usr/include/llvm-10")
-endif ()
```

Comment:
I think that these need to continue to be present for the people using windows

Reply:
That is still in the file actually, it just got moved somehow